# Service to save the current grasp learning model
---
int64 ret
string msg
